plugins/premium/comments/introduction_to_tiny_comments.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Introduction to Tiny Comments
+title_nav: Introduction
+description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
+keywords: comments commenting tinycomments
+---
+
+{% assign pluginname = "Comments" %}
+{% assign plugincode = "comments" %}
+
+## Contents
+
+* For help using comments in TinyMCE, see: [Using comments]({{site.baseurl}}/plugins/premium/comments/comments_end-user_docs/).
+* For an overview of the TinyMCE Comments plugin, see: [Overview](#overview).
+* For information on adding and configuring the comments plugin for TinyMCE, see: [Getting started with the Comments plugin - Selecting a mode](#gettingstartedwiththecommentsplugin-selectingamode).
+
+## Overview
+
+{{site.premiumplugin}}
+
+The Comments plugin provides the ability to start or join a conversation by adding comments to the content within the {{site.productname}} editor.
+
+### Collaborate on your projects within your content
+
+The Comments plugin provides:
+
+* A **user interface** to collaborate on content by creating and replying to comments.
+* A way to control the delete and resolve operations on a comment or comment thread.
+
+### Primary Comments Functions
+
+The Comments plugin allows the user to perform the following functions:
+
+* Create a comment
+* Edit a comment
+* Reply to a comment
+* Lookup a comment
+* Resolve a comment or comment thread
```

Comment:
This is incorrect, it only allows resolving a thread, not a single comment.

Reply:
Fixed in [`658e875` (#2007)](https://github.com/tinymce/tinymce-docs/pull/2007/commits/658e8758c071918697232fb55fb7df05f41c6960)